docs/modules/ROOT/examples/hazelcast-persistence.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  clusterSize: 3
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  version: '{page-latest-supported-hazelcast}-slim'
+  licenseKeySecret: hazelcast-license-key
+  persistence:
+    baseDir: "/data/hot-restart/"
+    clusterDataRecoveryPolicy: "FullRecoveryOnly"
+    pvc:
+      accessModes: ["ReadWriteOnce"]
+      requestStorage: 20Gi
```

Comment:
Currently, we do not have any cleanup or rotation mechanism so more warnings or detailed pieces of information about storage size config should be added.